k8s/webpackager/README.md

```diff
@@ -0,0 +1,308 @@
+# Overview
+
+Web Packager HTTP Server is an HTTP server built on top of Web Packager.
```

Comment:
I'd change this first para not to require foreknowledge of Web Packager or AMP Packager. Something like:

`Web Packager HTTP Server is an HTTP server that generates signed exchanges. By default, it is configured to enable privacy-preserving prefetch on referrers such as Google, by meeting the [requirements][] set by the Google SXG Cache. It functions like a reverse-proxy, fetching documents from a backend server, then optimizing and signing them before returning them to requestors.`

I think you can eliminate the "similar to AMP Packager" sentence because it's implied by the next para.

Reply:
Done.